server/src/termdb.js

```diff
@@ -463,3 +464,24 @@ async function LDoverlay(q, ds, res) {
 	})
 	res.send({ lst })
 }
+
+async function trigger_getDefaultGeneExpBins(q, ds, res) {
+	const tw = q.tw
+	const args = {
+		genome: q.genome,
+		dslabel: q.dslabel,
+		clusterMethod: 'hierarchical',
+		/** distance method */
+		distanceMethod: 'euclidean',
+		/** Data type */
+		dataType: 3,
+		genes: [{ gene: tw.term.gene }]
+	}
+	const data = await ds.queries.geneExpression.get(args)
+	let defaultBins
+	if (ds.queries.geneExpression.gene2bins[tw.term.gene])
+		defaultBins = { default: ds.queries.geneExpression.gene2bins[tw.term.gene] }
+	else defaultBins = { default: getDefaultGeneExpBins(ds, tw, data) }
```

Comment:
convert `data` into array of numbers and call `initBinConfig()`

Reply:
solved now